Wikipedia/UI-V5/WMFHomeSectionSchema.m

```diff
@@ -251,7 +259,28 @@ - (WMFHomeSection*)mainPageSection {
     return main;
 }
 
-- (WMFHomeSection*)continueReadingSection {
+- (nullable WMFHomeSection*)featuredArticleSection {
+    //Never show if we are not in english
+    if (![self.site.language isEqualToString:@"en"] && ![self.site.language isEqualToString:@"en-US"]) {
```

Comment:
yeah but featured article is a feature that is only enabled for en us, not en uk